### ISSUE TITLE
Wrong loss function computation for AV when loss=None

### ISSUE THREAD

**Issue by acano4i** [plnguyen2908/LASER_ASD#4]
Hi,
I encountered a critical bug in the lossAV class in loss_multi.py. The issue occurs in the forward method when labels is None.
Specifically, in the current code, line 18 reads:
```
if labels == None:
    return x
```

However, in the original LoCoNet code, it correctly handles this case with:
```
if labels == None:
    predScore = x[:, 1]
    predScore = predScore.t()
    predScore = predScore.view(-1).detach().cpu().numpy()
    return predScore
```

Returning x directly leads to incorrect predictions during inference. This breaks the expected behavior and causes downstream errors or invalid outputs.
Please update the code to reflect the original logic or let me know if you'd like a pull request.

**Comment by plnguyen2908** [comment]
Hi, thank you for your interest in our implementation.

The line of code that you cite actually comes from TalkNet's [loss.py](https://github.com/TaoRuijie/TalkNet-ASD/blob/main/loss.py). Particularly, x has shape (n,2) where x[:,0] returns the score for predicting not speaking and x[:1] returns the score for predicting speaking. They did that because in their [demoTalkNet.py](https://github.com/TaoRuijie/TalkNet-ASD/blob/main/demoTalkNet.py), they compare the score (which is x[:,1] in the code you cite) with 0. You can find that in line 274.

We will change that to match the original implementation. However, we just want to mention that it won't lead to incorrect predictions if you use it correctly. You can either compare x[i,1] with x[i,0] or return x[:,1] and compare with 0. Either way works.
